install_config/topics/using_gcp_load_balancer.adoc

```diff
@@ -0,0 +1,119 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='gce-load-balancer_{context}']
+= Using the GCP external load balancer as a service
+
+{product-title} can leverage the GCP load balancer capacity by exposing services
+externally using a `LoadBalancer` service. {product-title} creates the load
+balancer in GCP and creates the proper firewall rules.
+
+The following is an example `LoadBalancer` service:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: hello-openshift
+  name: hello-openshift-external
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - name: port-1
+    nodePort: 30714
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  - name: port-2
+    nodePort: 30122
+    port: 8888
+    protocol: TCP
+    targetPort: 8888
+  selector:
+    app: hello-openshift
+    deploymentconfig: hello-openshift
+  sessionAffinity: None
+  type: LoadBalancer
+----
+
+.Procedure
+
+. Create a new application:
++
+[source,bash]
+----
+$ oc new-app openshift/hello-openshift
+----
+
+. Expose the load balancer service:
++
+[source,bash]
+----
+$ oc expose dc hello-openshift --name='hello-openshift-external' --type='LoadBalancer'
+----
+
+. To verify that the service has been created:
```

Comment:
We didn't create the service? Is this the command to confirm that it's been exposed?

Reply:
the service was created with the oc expose dc command before, the oc get svc is to confirm it has been exposed